backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Purchase_InOut_Details.sql

```diff
@@ -23,95 +23,89 @@ CREATE OR REPLACE FUNCTION de_metas_endcustomer_fresh_reports.Docs_Purchase_InOu
             )
 AS
 $$
-SELECT Attributes,
-       Name, -- product
-       SUM(HUQty)       AS HUQty,
-       HUName,
-       SUM(MovementQty) AS MovementQty,
-       UOMSymbol,
-       StdPrecision,
-       QualityDiscountPercent,
-       QualityNote,
-       isInDispute,
-       iol.Description,
-       bp_product_no,
-       bp_product_name,
-       MAX(iol.line)    AS line
+SELECT *
```

Comment:
pls, do not format
Is very difficult to see what you changed

Reply:
done